src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ShipmentRepository.php

```diff
@@ -41,6 +42,21 @@ public function findOneByOrderId($shipmentId, $orderId): ?ShipmentInterface
         ;
     }
 
+    public function findOneByOrderTokenAndChannel($shipmentId, string $tokenValue, ChannelInterface $channel): ?ShipmentInterface
+    {
+        return $this->createQueryBuilder('o')
+            ->innerJoin('o.order', 'ord')
```

Comment:
`'ord'` -> `'order'`? 😅

Reply:
naming it `order` is causing problems when I try to implement it, probably it's restricted by `querybuilder`, what do you think about naming it for example `orders` instead?